### DISCUSSION TITLE
using apostrophe for static site?

### DISCUSSION THREAD

**Discussion by nagotown** [apostrophecms/apostrophe#3835] [Q&A] [answered]
is it possible to use apostrophe for a static website? it's rather expensive to host dynamic sites and i'm fine just using the cms locally.
i can code my own site but i like the structure of visual website editors, since my sites are rather simple.
if not, could someone recommend any alternatives? thanks.

**Comment by boutell** [comment, marked as answer]
Hi @nagotown,

I'd say the cheapest solid hosting option for ApostropheCMS is probably an Ubuntu Linux "droplet" from Digital Ocean. At $12/month, it's very reasonable.

Otherwise I suppose you could consider using a site-mirroring tool to create a static copy of it while running it locally.
